docs/reference.rst

```diff
@@ -26,7 +26,7 @@ API Reference
 
     :param data: :term:`SAML Response` as extracted from the HTTP form field ``SAMLResponse``.
     :param certificate: Certificate or collection of certificates used by the :term:`Identity Provider`.
-    :param expected_audience str: :term:`Audience` of your :term:`Identity Provider`.
```

Comment:
Kind of curious about what this 'str:' was doing before... it looks like it was out of sync with the surrounding text at least format wise...

Reply:
yea, that's why I removed it. the types are actually handled by `.. autofunction::`. since the function has type-hints, there is no need to manually type them in the docs.